PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/toolkit/intellij/function/action/DeployFunctionAppAction.java

```diff
@@ -35,10 +39,11 @@
 
     private final AzureFunctionSupportConfigurationType functionType = AzureFunctionSupportConfigurationType.getInstance();
 
-    private Project project;
-    private FunctionAppNode functionNode;
+    private final Project project;
+    private final FunctionAppNode functionNode;
 
     public DeployFunctionAppAction(FunctionAppNode functionNode) {
+        super();
```

Comment:
it seems unnecessary.  and it also fine.

Reply:
to resolve ide compile warning